### PR TITLE
Add backwards-compatibility notice.

### PR DESCRIPTION
**Pull request by sbialkowski-r7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 Allows you to send logs to the [Insight Platform](https://www.rapid7.com/products/)
 (or Logentries) account from Node.js.
 
+This client is not backwards-compatible with [Le_Node](https://github.com/rapid7/le_node).
```
Comment (rjacobs-r7) [inline README.md]:
```suggestion
This client is not backwards-compatible with [Le_Node](https://github.com/rapid7/le_node) and will require you to make some modifications to your implementation.
```

Comment (sbialkowski-r7) [reply README.md]: Would be ideal, but there is so much that's changed.
Every package updated, how we generate Transports etc.
Don't think we'd ever hit everything and I can't imagine an easy switchover.

**Comment by sbialkowski-r7** [comment]
Migration guide here:
https://github.com/rapid7/r7insight_node/wiki/2.0-Migration-Guide

File: README.md [hunk 2]
```diff
@@ -5,6 +5,8 @@
 Allows you to send logs to the [Insight Platform](https://www.rapid7.com/products/)
 (or Logentries) account from Node.js.
 
+This client is not backwards-compatible with [Le_Node](https://github.com/rapid7/le_node) and will require you to make some modifications to your implementation.
```
Comment (rjacobs-r7) [inline README.md]:
```suggestion
This client is not backwards-compatible with [Le_Node](https://github.com/rapid7/le_node) and will require you to make some modifications to your implementation. An upgrade guide can be found on [the wiki](https://github.com/rapid7/r7insight_node/wiki/2.0-Migration-Guide).
```

**Pull request merged**